string table_name
string message
---
string echo